.changeset/cute-worlds-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@solid-design-system/components': patch
+---
+
+Improve `sd-flipcard` focus management for better accessibility.
```

Comment:
Didn't ask Manuel in addition to get rid of flipping but just "dissolving" in addition?

Reply:
He did, but in the ticket it's mentioned that this is currently a nice to have since we need to comply with WCAG level AA, not level AAA.